sdcm/nemesis.py

```diff
@@ -1145,6 +1145,17 @@ def disrupt(self):
         self.call_random_disrupt_method()
 
 
+class GeminiChaosMonkey(Nemesis):
+
+    @log_time_elapsed_and_status
+    def disrupt(self):
+        # Limit the nemesis scope to use with gemini
+        # - StopStartMonkey
+        # - RestartThenRepairMonkey
+        self.call_random_disrupt_method(disrupt_methods=['disrupt_stop_start_scylla_server',
```

Comment:
It's ok for now, but:
1. We need to stop creating these lists "manually" and have it built by a property.
2. We need a task in Trello to extend this list.

Reply:
it could be the same as it is done for ParallelNemesis Disruptive and NoneDisruptive. For example flag name : run_with_gemini
Task is added to Trello